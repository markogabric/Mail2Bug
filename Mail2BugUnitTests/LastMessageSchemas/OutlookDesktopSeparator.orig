<html>
<body>
<div class="wrappingBothMessageAndReply">
  <b><p class="customStyling">This is random text with some custom styling and outlook-generated elements.<o:p></o:p></p></b>
  <div>
    <div style="border:none;border-top:solid #E1E1E1 1.0pt;padding:3.0pt 0in 0in 0in">
      <p class="customStyling">
        <b>Old emails from: field and subject</b>
      </p>
    </div>
  </div>
  <p class="customStyling"><o:p>&nbsp;</o:p></p>
  <div>
      Something after the containing div
  </div>
</div>
</body>
</html>