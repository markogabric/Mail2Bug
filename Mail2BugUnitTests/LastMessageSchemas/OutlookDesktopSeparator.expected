<html><head></head><body>
<div class="wrappingBothMessageAndReply">
  <b><p class="customStyling">This is random text with some custom styling and outlook-generated elements.<o:p></o:p></p></b>
  </div></body></html>